CHANGELOG-prerelease.md

```diff
@@ -1,421 +1,22 @@
-## Prerelease 56 ( 2020-08-26 )
```

Comment:
@starryeyez024 There is a CHANGELOG-1.x.md in the repo that we should be using going forward.

Reply:
thanks, updated